modules/apps/web-experience/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp

```diff
@@ -329,8 +331,13 @@ RoleSearchTerms searchTerms = (RoleSearchTerms)roleSearchContainer.getSearchTerm
 					<liferay-ui:search-container-column-text
 						href="<%= definePermissionsHREF %>"
 						name="role"
-						value="<%= role.getTitle(locale) %>"
-					/>
+					>
+						<%= role.getTitle(locale) %>
+
+						<c:if test="<%= layout.isPrivateLayout() && name.equals(RoleConstants.GUEST) %>">
+							<liferay-ui:icon-help message="guest-permissions-inherited" />
+						</c:if>
+					</liferay-ui:search-container-column-text>
```

Comment:
@pavel-savinov this fix looks good! However, I think this commit needs to be either re-named or broken into two commits.  The commit message implies you just added a tooltip, but the code shows you also adjusted when the Guest role is filtered out of the list.  I think it's important that that change is either documented in the commit message, or placed in a separate commit altogether (my preference).  

Could you please make that change and re-send to me?  Thanks!